std/array.d

```diff
@@ -263,8 +263,9 @@ unittest
 }
 
 /**
-Returns a newly allocated associative array out of elements of the input range,
-which must be a range of tuples (Key, Value).
+Params: r = An input range of tuples of keys and values.
```

Comment:
On Wed, Jan 07, 2015 at 09:49:09AM -0800, Andrej Mitrović wrote:
> Quick question: Will the docs be generated properly with a lack of a blank line between `Params` and `Returns` (and `Returns and `See Also`) sections?

Yep, see the preview here: http://eusebeia.dyndns.org/~hsteoh/tmp/web/phobos-prerelease/std_array#.assocArray